gladier/base.py

```diff
@@ -0,0 +1,26 @@
+from .client import GladierClient as GladierBaseClient
+
+
+##not sure how to update this.
+# class GladierBaseClient(object):
+
+#     container_url = None
+#     container_type = None
+#     container_location = None
+#     container_flags = None
+
+class GladierBaseTool(object):
+    """Gladier Defaults defines a common method of tying together
+    flows, funcx-functions, and default inputs for starting a flow."""
+
+    flow_definition = None
+    flow_input = dict()
+    funcx_functions = []
```

Comment:
Should Base Tools include an optional `funcx_container` that is `None` by default (simply runs the function without a container if None) but can be configured by the BaseTool if desired?

Reply:
That was my idea.
For now the way that funcx have been dealing with the containers is the following. 
`tomo_cont_id = fxc.register_container(location=os.path.join(conf['cont_dir'],container_name), container_type='singularity') 
download_cont_fxid = fxc.register_function(download_cont)
recon_fxid = fxc.register_function(tomo_recon, container_uuid=tomo_cont_id)`

so basically. if container_uuid is None, it does not register anything and treats as a normal funcx function. The problem is this is not on the master branch yet.